apis/core/bootstrap/shapes/table.ts

```diff
@@ -27,8 +27,8 @@ const commonProperties = turtle`
     ${sh.path} ${cc.identifierTemplate} ;
     ${sh.minCount} 1 ;
     ${sh.maxCount} 1 ;
-    ${sh.defaultValue} "" ;
```

Comment:
For some reason, `dash:editor` was not taken into account when `sh:defaultValue` was specified. Now I have a feeling we'll run into issues without this default value?

Reply:
Yes, without `sh:default` we would get validation errors unless the user interacts with the field. Not good.

The problem is that the preferred editor would get removed if there is no metadata about it.

Fixed soon by https://github.com/hypermedia-app/shaperone/pull/84.

Reply:
👍 Can wait on the shaperone PR

Reply:
I upgraded shaperone but it seems like it still doesn't select the proper editor when creating a new table

Reply:
Yes, my mistake